plugin-developer-learning-pathway/07-Custom-Database-Tables/01-Creating-and-Managing-Custom-Tables.md

```diff
@@ -132,17 +173,40 @@ When a user deactivates or uninstalls your plugin, it’s good practice to clean
 
 In most cases, plugin data is removed during uninstallation because it indicates the user no longer needs the plugin. As you learned in Module 1, you can use the `register_uninstall_hook()` function to do this.
 
-You can execute the SQL query to delete the table by using the `$wpdb` global variable, which we'll learn more about in the next lesson.
+You can execute the SQL query to delete the table by using the `$wpdb` global variable, which you'll learn more about in a future lesson.
 
 ```php
-register_uninstall_hook( __FILE__, 'my_plugin_delete_table' );
-function my_plugin_delete_table() {
+register_uninstall_hook( __FILE__, 'wp_learn_custom_table_delete_submissions_table' );
+function wp_learn_custom_table_delete_submissions_table() {
 	global $wpdb;
-	$wpdb->query( "DROP TABLE IF EXISTS {$wpdb->prefix}my_plugin_table" );
-	delete_option( 'my_plugin_db_version' );
+	$wpdb->query( "DROP TABLE IF EXISTS {$wpdb->prefix}submissions" );
+	delete_option( 'wp_learn_custom_table_db_version' );
 }
 ```
 
+## A note on custom table names
+
+In the examples in this lesson, you've created a table called `submissions`, prefixed using the `$wpdb->prefix`, so the table name ends up being `wp_submissions`.
+
+Generally, you will want to also include a prefix in the table name following a similar structure as the [prefixing method used to avoid naming collisions](https://developer.wordpress.org/plugins/plugin-basics/best-practices/#procedural-coding-method) in PHP. 
+
+This makes the table unique to your plugin, and avoids any table conflicts.
+
+So to improve the above examples, you might include a vendor prefix like `wpl` (instead of `wp_learn`).
+
+```php
+	$sql = "CREATE TABLE {$wpdb->prefix}wpl_submissions (
+	id mediumint(9) NOT NULL AUTO_INCREMENT,
+	time datetime DEFAULT '0000-00-00 00:00:00' NOT NULL,
+	name tinytext NOT NULL,
+	message text NOT NULL,
+	url varchar(55) DEFAULT '' NOT NULL,
+	PRIMARY KEY  (id)
+	) {$charset_collate};";
+```
+
+This will create a table with the name `wp_wpl_submissions`.
+
 ## Conclusion
 
-Now that we know how to create, update, and delete a custom table in the WordPress database, let's see how to read and write data within the table. We'll cover these concepts in the next lesson, "Interacting with Custom Tables".
+For more information on working with custom tables, make sure to read the [Creating Tables with Plugins section of WordPress Plugin Handbook](https://developer.wordpress.org/plugins/creating-tables-with-plugins/). It's also useful to read the class reference for the `WPDB` class, as well as the function reference for `dbDelta`.
```

Comment:
>the `WPDB` class

should be

> the `wpdb` class

Can we also link `the class reference` to https://developer.wordpress.org/reference/classes/wpdb/ and `the function reference` to https://developer.wordpress.org/reference/functions/dbdelta/ ?

I know we linked to them earlier in the lesson, but it feels confusing to me as a user that wants to read more. I feel uncertain about what these are referring to, and I can't take immediate action to continue reading from here in the lesson.

Reply:
100% Agree, reiterating links back to developer docs and function references is my preferred way to wrap up a lesson.